Wireframe/css/style.css

```diff
@@ -0,0 +1,191 @@
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 400;
+  src: local("Lato Regular"), local("Lato-Regular"),
+    url("../fonts/lato-v15-latin-regular.woff2") format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../fonts/lato-v15-latin-regular.woff") format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* lato-700 - latin */
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 700;
+  src: local("Lato Bold"), local("Lato-Bold"),
+    url("../fonts/lato-v15-latin-700.woff2") format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../fonts/lato-v15-latin-700.woff") format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* lato-900 - latin */
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 900;
+  src: local("Lato Black"), local("Lato-Black"),
+    url("../fonts/lato-v15-latin-900.woff2") format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../fonts/lato-v15-latin-900.woff") format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+html,
+body {
+  height: 100%;
+}
+
+html {
+  height: 100%;
+  scroll-behavior: smooth;
+  box-sizing: border-box;
+}
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  height: 100%;
+  font-family: lato;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 1.2;
+  font-weight: normal;
+  color: black;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.page {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.main {
+  flex: 1 0 auto;
+}
+
+.header {
+  margin: 0 auto;
+}
+
+.title {
+  text-align: center;
+}
+
+.wrapper-article {
+  max-width: 1060px;
+  margin: 0 auto;
+  display: flex;
+
+  flex-direction: column;
+}
+
+
+img {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  margin-bottom: 15px;
+}
+
+.articles {
+  margin: 20px 0;
+  display: flex;
+  justify-content: space-around;
+}
+
+.article {
+  padding: 20px;
+}
+
+.articles:first-child {
+  background-color: aqua;
+}
+
+.article_btn {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 26px;
+
+  margin-right: 38px;
+
+  width: 161px;
+  height: 50px;
+
+  color: #e06733;
+
+  box-sizing: border-box;
+
+  border: 1px solid #e06733;
+  border-radius: 5px;
+
+  -webkit-transition-duration: 0.4s;
+  transition-duration: 0.4s;
+  cursor: pointer;
+  background-color: #010600;
+  transition: all 0.3s ease-in;
+}
+
+.article_btn:hover {
+  background-color: #4caf50;
+  color: white;
+}
+
+.main_article {
+  display: block;
+  padding: 20px;
+}
+
+.page {
+  overflow: hidden;
+}
+
+.footer {
```

Comment:
In our session we explored using position: fixed to fix this footer to the bottom and then we padded the main element to move the buttons out of the way.